src/fragments/plugins.ts

```diff
@@ -10,9 +10,20 @@ export const configurationItemFragment = gql`
   }
 `;
 
-export const pluginConfigurationFragment = gql`
+export const pluginConfigurationBaseFragment = gql`
+  fragment PluginConfigurationBaseFragment on PluginConfiguration {
+    active
+    channel {
+      id
+      name
+      slug
+    }
+  }
+`;
+
+export const pluginConfigurationExtendedFragment = gql`
   ${configurationItemFragment}
-  fragment PluginConfigurationFragment on PluginConfiguration {
+  fragment PluginConfigurationExtendedFragment on PluginConfiguration {
```

Comment:
This extended fragment could use `pluginConfigurationBaseFragment` as a base:
```
${configurationItemFragment}
${pluginConfigurationBaseFragment}
fragment PluginConfigurationExtendedFragment on PluginConfiguration {
  ...PluginConfigurationBaseFragment
  configuration {
      ...ConfigurationItemFragment
  }
}
```

Reply:
That's been changed now